src/components/button/templates/component/logos/venmo_blue.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="57px" height="11px" viewBox="0 0 57 11" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Can you make sure you're getting all the same options from the other SVGs, like `preserveAspectRatio`. That will help with cross-browser compatibility (spent a lot of time banging my head against the desk to figure those out and want to make sure you don't have to suffer too!)